packages/esm-form-engine-app/translations/en.json

```diff
@@ -4,9 +4,5 @@
   "loading": "Loading",
   "or": "or",
   "thisList": "this list",
-  "tryAgainMessage": "Try opening another form from",
-  "emptyIcon": "Empty icon",
-  "errorLoadingForm": "An error occurred while loading the form",
-  "errorLoadingFormMessage": "An error occurred while loading the form",
-  "missingJSONSchema": "The specified form is missing JSON schema"
```

Comment:
Why the updates on the translation keys?

Reply:
That is exactly what i was going to also ask. 😊